docs/user_guide/method/writing_custom_methods.rst

```diff
@@ -0,0 +1,528 @@
+.. _writing_custom_methods:
+
+======================
+Writing Custom Methods
+======================
+
+The power of mrsimulator lies in the method object. Most time-domain NMR simulation
+software requires users to walk through all aspects of a pulse sequence, but mrsimulator
+preforms calculations in the frequency-domain and only needs descriptions of the spectral
+dimensions being simulated. For a more in-depth discussion on how a method defined
+spectral dimensions, see the :ref:`method_documentation`.
+
+On this page, we will illustrate how to write custom methods by simulating progressively
+more complicated NMR experiments on :math:`RbNO_3`. First lets import the necessary
```

Comment:
Change RbNO3 math directives to :math:`\text{RbNO}_3` so characters are not slanted